src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_cthun.cpp

```diff
@@ -448,33 +419,21 @@ struct boss_cthun : public BossAI
         }).Schedule(30s, [this](TaskContext context)
         {
             if (Creature* eye = instance->GetCreature(DATA_EYE_OF_CTHUN))
-            {
                 eye->AI()->DoAction(ACTION_SPAWN_EYE_TENTACLES);
-            }
 
-            context.Repeat(30s);
+            context.Repeat();
```

Comment:
why did you drop the 30s?

Reply:
Because it is 30s in the first call so it’s only necessary to repeat it